Eastern-Research-Group/mywaterway
# HMW-3 Fixed an issue with the background image for the HMW widget not…

## Related Issues:
* [HMW-3](https://jira.epa.gov/browse/HMW-3)

## Main Changes:
* Fixed an issue with the HMW widget background image not displaying on the dev site.
* Added a `communityIframeTestpage.html` page. This page shows the HMW widget in 4 different sizes.
  * I modified the `communityWidget.html` file so the widget looks better at different sizes.

## Steps To Test:
1. Navigate to http://localhost:9090/widgets/communityIframeTestpage.html
2. Verify the 4 widgets display
3. Verify the background image displays on all 4 widgets 

## TODO:
- [ ] The images need to be moved to the production server (i.e., https://www.epa.gov/sites/production/files/2021-10/boating.jpg) and the code needs to be updated to point to the production urls.
